### PR TITLE
PUBDEV-4778, PUBDEV-5052: Fix predicting with xgboost

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

**Comment by michalkurka** [comment]
@h2o-ops please test!

File: h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package hex.tree.xgboost;
 
+import com.sun.jna.Memory;
```
Comment (Pscheidl) [inline h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java]: Unused import

**Comment by mmalohlava** [comment]
I would look at failing test - it is directly connected to to this PR.

**Comment by michalkurka** [comment]
@h2o-ops please test!

**Pull request merged**